src/apis/index.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
+axios.defaults.baseURL = 'https://api.github.com';
+
+const API = axios;
+
+export function getRepos(keyword) {
+  return API.get(`/users/${keyword}/repos`).then(({ data }) => {
+    const owner = data.length && data[0].owner.login;
+
+    const repos = data.map(
+      ({
+        id,
+        name,
+        description,
+        html_url: url,
+        stargazers_count: starCount,
+      }) => ({
+        id,
+        name,
+        description,
+        url,
+        starCount,
+      }),
+    );
+
+    return {
+      keyword: owner,
+      repos,
```

Comment:
eslint에서 airbnb 룰을 사용하셨던데 린트에서 아래와 같이 하라고 제안 안했었나요??
> airbnb 룰을 사용하면 다음과 같이 하라 했던 기억이 있어서요.

```suggestion
      repos,
      keyword: owner,
```

Reply:
음 저는 못봤는데 둘이 어떤 차이가 있을까요?! 무슨 룰인지 궁금하네요 !! 
prettier 플러그인도 같이 사용중이라 prettier 플러그인을 지워봐도 뜨지 않아요 ㅜㅜ

Reply:
음 저는 못봤는데 둘이 어떤 차이가 있을까요?! 무슨 룰인지 궁금하네요 !!
prettier 플러그인도 같이 사용중이라 prettier 플러그인을 지워봐도 뜨지 않아요 ㅜㅜ

Reply:
정렬 순서와 관련된 룰로 알고 있는데요. Airbnb 룰에는 

key와 value가 같아 shorthand로 사용 가능하면 shorthand로,
shorthand가 가능한 것과 불가능한 것이 혼재 되어있을 경우 shorthand 먼저 명시하는 룰이 있어요!

https://github.com/airbnb/javascript#objects--grouped-shorthand